service/assessment/assessment_test.go

```diff
@@ -459,7 +460,7 @@ func TestAssessmentResultHooks(t *testing.T) {
 			// To test the hooks we have to call a function that calls the hook function
 			gotResp, err := s.AssessEvidence(tt.args.in0, tt.args.evidence)
 
-			// wait for all hooks (6 metrics * 2 hooks)
+			// wait for all hooks (2 metrics * 2 hooks)
```

Comment:
@lebogg how many metrics should be evaluated in the AssessEvidence for this testcase? 6 or 2?